src/Jackett.Common/Indexers/TorrentHeaven.cs

```diff
@@ -32,7 +32,7 @@ private new ConfigurationDataCaptchaLogin configData
         public TorrentHeaven(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps)
             : base(name: "TorrentHeaven",
                    description: "A German general tracker.",
-                   link: "https://torrentheaven.myfqdn.info/",
```

Comment:
please add a legacysitelinks for the old domain so that user's configs can get auto-updated to the new sitelink without them having to manually update their configs.
here is an example:
https://github.com/Jackett/Jackett/blob/c830a59d58e1d0b63decc76b84f3c1588aec9ed7/src/Jackett.Common/Indexers/ImmortalSeed.cs#L24-L26